k8s/operator/values.yaml

```diff
@@ -70,12 +79,16 @@ sdp:
   ## @param sdp.gitOperator.git.sshPort Alternate ssh port for the git repository (if not well-known port)
   ## @param sdp.gitOperator.git.strictHostKeyChecking Enable StrictHostKeyChecking when cloning repository via SSH
   gitOperator:
-    version: v18
     logLevel: info
     timeout: 30
     targetTags: []
     dryRun: true
     secret: ""
+    securityContext:
+      runAsUser: 1000
+      runAsGroup: 1000
+      fsGroup: 1000
```

Comment:
Do we create these uid and gid in the Dockefile? Or are they created automatically?